src/main/java/org/randywebb/wlts/Main.java

```diff
@@ -30,27 +30,38 @@
 * @author randyw
 *
 */
-public class Main {
+public final class Main {
 
-    /** Can be used for logging debugging messages */
+    /** Can be used for logging debugging messages. */
     private static Logger log = LoggerFactory.getLogger(Main.class);
 
+    /** default constructor. */
+    private Main() {
```

Comment:
Why do we need to call out an empty default constructor?
I don't think we ever instantiate this class anyway as there's no reason to.

Reply:
The private default constructor prevents instantiation. Methods without docs cause an error. I can make the missing javadoc a warning. Any other suggestions?

Reply:
It makes sense that we wouldn't want to instantiate the Main class.